install.sh

```diff
@@ -263,12 +281,14 @@ if [ "$upgrade" -eq 1 ] ; then
 	fi
 fi
 
-if [ "$silent_install" -ne 1 ] ; then 
+if [ "$silent_install" -ne 1 ] ; then
 	echo "$line"
 	echo -e "\t$(gettext "Please choose what you want to install")"
 	echo "$line"
 fi
 
+export upgrade
```

Comment:
$upgrade is not already exported ?

Reply:
nope, "inst_upgrade_dir" and other variables are exported here :
https://github.com/centreon/centreon/pull/8396/files/aa6a8ba9250af654f7cd1f71316eb2848e354226#diff-3fbb47e318cd8802bd325e7da9aaabe8R168
but these aren't directly relevant with the upgrade process